### ISSUE TITLE
[Bug] Buttons don't work on 'How satisfied are you with Firefox Relay?' feedback banner

### ISSUE THREAD

**Issue by pendantry** [mozilla/fx-private-relay#1564]
As requested by @Vinnl [here](https://github.com/mozilla/fx-private-relay/issues/1563#issuecomment-1048588155), I'm opening a new issue related to Firefox Relay.

At [my 'profile' URL](https://relay.firefox.com/accounts/profile/) I've recently been presented with a banner at the top:

![image](https://user-images.githubusercontent.com/77969662/155338388-92dce118-a6c7-4aeb-a271-381e09d63475.png)

– but the satisfaction buttons don't work. I've tried 'Neutral', 'Dissatisfied' and 'Very Dissatisfied' (and refuse to try the other two in case they work!)

@Vinnl asked me to include any output I see [in the web console](https://developer.mozilla.org/en-US/docs/Tools/Web_Console#opening_the_web_console), so here 'tis:

![image](https://user-images.githubusercontent.com/77969662/155338001-232a686d-26d4-49a8-8ea2-9ee44d72b651.png)

A thought: is it perhaps possible that one of the other Firefox add-ons I have installed is interfering with Firefox Relay? The ones I have installed are:

AdBlocker Ultimate
Enhancer for YouTube
Facebook Container
Firefox Relay

(Hmm... I also used to have DuckDuckGo Privacy Essentials but it seems to have inexplicably vanished. Wonder what happened to that? I've now added it back in.)

**Comment by Vinnl** [comment]
Ah, I managed to reproduce this: we've got code that hides the survey for people with adblockers, but the script that contains that code _gets blocked by those same blockers_ I'll prepare a fix shortly.

PS. Thanks for reporting!
